component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml

```diff
@@ -900,4 +900,10 @@
         <createSequence sequenceName="SEQ_SOC_LABELS_ID" startValue="1"/>
     </changeSet>
 
+    <changeSet author="social" id="1.0.0-91" dbms="mysql">
+        <sql>
+            ALTER TABLE SOC_ACTIVITY_TEMPLATE_PARAMS MODIFY COLUMN TEMPLATE_PARAM_KEY NVARCHAR(255) BINARY;
```

Comment:
what about emojis in TEMPLATE Params like kudos message .... does it work on MySQL with this change without `utf8mb4_unicode_ci` ?

Reply:
@boubaker But this changes only the param **key** column (It's value always in English as its the name of the param and no its value isn't it ?) + it juts changes the collation to a binary one but not the collation type.

Reply:
ok, I see, thanks for explanation, ok for me.